frontend/src/metabase/query_builder/components/QueryVisualization.jsx

```diff
@@ -117,9 +128,16 @@ export const VisualizationDirtyState = ({
   hidden,
 }) => (
   <div
-    className={cx(className, "Loading flex flex-column layout-centered", {
-      "Loading--hidden pointer-events-none": hidden,
-    })}
+    className={cx(
+      className,
+      QueryBuilderS.Loading,
+      CS.flex,
+      "flex-column layout-centered",
+      {
+        [QueryBuilderS.LoadingHidden]: hidden,
+        "pointer-events-none": hidden,
```

Comment:
I'm not sure if we should keep the class in the same line as before or separated like this.

```suggestion
        [`${QueryBuilderS.LoadingHidden} pointer-events-none`]: hidden,
```

I think putting them on the same line makes them resemble the previous version more. It's only slightly better imo, so it's fine to leave them as is. I'll leave it to your call.

Reply:
even better. 👍